src/Lykke.Job.BlockchainOperationsExecutor/Workflow/CommandHandlers/BroadcastTransactionCommandsHandler.cs

```diff
@@ -39,20 +39,24 @@ public class BroadcastTransactionCommandsHandler
             {
                 case TransactionBroadcastingResult.Success:
                     break;
+                
                 case TransactionBroadcastingResult.AlreadyBroadcasted:
-                    _log.WriteInfo
+                    await _log.WriteInfoAsync
                     (
+                        nameof(BroadcastTransactionCommandsHandler),
                         nameof(BroadcastTransactionCommand),
-                        command.OperationId,
+                        command.ToString(),
```

Comment:
.ToJson() should be used here instead of .ToString(), shouldn't it?

Reply:
Yes, sure